tests/client/amo/components/TestOverallRating.js

```diff
@@ -1,31 +1,203 @@
+/* eslint-disable import/prefer-default-export */
 import React from 'react';
 import { findDOMNode } from 'react-dom';
 import {
   findRenderedComponentWithType,
   renderIntoDocument,
+  Simulate,
 } from 'react-addons-test-utils';
 
-import OverallRating from 'amo/components/OverallRating';
+import { mapDispatchToProps, mapStateToProps, OverallRatingBase }
+  from 'amo/components/OverallRating';
```

Comment:
We've generally been breaking in the parens rather than before `from`.

```js
import {
  mapDispatchToProps,
  mapStateToProps,
  OverallRatingBase,
} from 'amo/components/OverallRating';
// or
import {
  mapDispatchToProps, mapStateToProps, OverallRatingBase,
} from 'amo/components/OverallRating';